.github/workflows/rust.yml

```diff
@@ -10,6 +10,13 @@ jobs:
     - name: Check fmt
       run: cargo fmt -- --check
 
+  msrv:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dtolnay/rust-toolchain@1.54.0
+    - run: cargo check --all-features
```

Comment:
I think most crates make their MSRV promise for the default build or at least only for the default dependencies. `--all-features` will enable all optional dependencies which might be a harder promise to keep?

Reply:
Agree, I think, we should also test with `cargo check` and set that version as MSRV.